### PR TITLE
150: Show discount amount on Checkout screen

### PR DESCRIPTION
**Pull request by MiteshiOS**
## Notes

- Added logical updates for showing the discount amount on the checkout screen

## Visuals

https://user-images.githubusercontent.com/17427703/181915333-63801e66-2b4e-4e7a-83dd-63b9bcb2288e.mp4

### CODE REVIEW

File: archiv333/View Controllers/Checkout/CheckoutViewController.swift [hunk 1]
```diff
@@ -192,15 +192,23 @@ extension CheckoutViewController: UITableViewDelegate, UITableViewDataSource {
...
-                DispatchQueue.main.async { self.present(viewControllerToPresent: Utils.alert(title: "Oops", message: "You've already applied a promo code to this order.\n\nPromo codes cannot be combined.", buttonText: "Dismiss")) }
+
+                //DispatchQueue.main.async { self.present(viewControllerToPresent: Utils.alert(title: "Oops", message: "You've already applied a promo code to this order.\n\nPromo codes cannot be combined.", buttonText: "Dismiss")) }
+
+                let alert = Utils.alert(title: "Oops", message: "You've already applied a promo code to this order.\nPromo codes cannot be combined.\n\nDo you wants to remove this code ?", buttonText: "Yes", buttonStyle: .destructive, showDismissButton: true, clickHandler:  { _ in
+                    Archiv.Session.promoCode = nil
+                    Archiv.Session.totalAdjustment = 0.00
+                    DispatchQueue.main.async { self.tableView.reloadData() }
+                })
+                DispatchQueue.main.async { self.present(viewControllerToPresent: alert) }
```
Comment (tejen) [inline archiv333/View Controllers/Checkout/CheckoutViewController.swift]: 🔍 Does this need to be explicitly in the main thread? To present the alert here. @MiteshiOS

**Review by tejen — CHANGES_REQUESTED** [review]

File: archiv333/View Controllers/Checkout/CheckoutViewController.swift (hunk 1 above)
Comment (MiteshiOS) [reply archiv333/View Controllers/Checkout/CheckoutViewController.swift]: @tejen Oho ya it's not needed. Let me correct it.

**Review by tejen — APPROVED** [review]

**Comment by tejen** [comment]
[Show discount amount on Checkout screen](https://trello.com/c/braR8TD6/150-show-discount-amount-on-checkout-screen)

**Comment by tejen** [comment]
[Show discount amount on Checkout screen](https://trello.com/c/braR8TD6/150-show-discount-amount-on-checkout-screen)

**Pull request merged**